reactioncommerce/reaction-next-starterkit
# feat: 353 nnnnat status labels

Resolves #353 
Impact: **minor**
Type: **feature|bugfix**

## Issue
Checkout Action labels needed to change depending on their status. v0.41.1 of `@rc/components` now support "status" labels for `CheckoutActions`.

## Solution
* Updated to v0.41.1 of component library
* Updated checkout `actions` to have `id` and `activeLabel, completeLabel, incompleteLabel` props.
 * Fixed a bug with loading locales data in the starterkit.

## Breaking changes
N/A

## Testing
1. As a customer add an item to cart and checkout as a guest.
2. Verify each `CheckoutAction`  label against the [design](https://zpl.io/2y7D0Nn) during the checkout steps.
3. Verify the `AddressForm` country field is loading as a `Select` with country data.
